tests/lib/Stubs/ItemGroupList.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzRecommendationClient\Tests\Stubs;
+
+use ArrayIterator;
+use Ibexa\Contracts\Personalization\Value\ItemGroupListInterface;
+use IteratorAggregate;
+use Traversable;
+
+/**
+ * @implements IteratorAggregate<\Ibexa\Contracts\Personalization\Value\ItemGroupInterface>
```

Comment:
Reported the issue (duplicated phpdoc without parent type check) in phpstan: https://github.com/phpstan/phpstan/discussions/5221